test/Dialect/FIRRTL/wire-dft.mlir

```diff
@@ -34,14 +35,14 @@ firrtl.circuit "Simple" {
   // CHECK: firrtl.module @A(in %test_en: !firrtl.uint<1>)
   firrtl.module @A() {
     %in, %test_en, %en, %out = firrtl.instance eicg @EICG_wrapper(in in: !firrtl.clock, in test_en: !firrtl.uint<1>, in en: !firrtl.uint<1>, out out: !firrtl.clock)
-    // CHECK: firrtl.connect %eicg_test_en, %test_en : !firrtl.uint<1>, !firrtl.uint<1>
+    // CHECK: firrtl.strictconnect %eicg_test_en, %test_en : !firrtl.uint<1>
```

Comment:
The backport picks up a move to `emitConnect` instead of creating `ConnectOp`s directly, but this should be very safe in terms of WireDFT changes.